modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordExecManager.kt

```diff
@@ -55,14 +55,12 @@ class MirrordExecManager(private val service: MirrordProjectService) {
             service
                 .notifier
                 .notification(
-                    "mirrord plugin was unable to display the target selection dialog. " +
```

Comment:
I think it still will lead to the same issue, no? like the Open button wouldn't work because it's null?

Reply:
It's ok now. If the path is null, the button is not added to the notification, see lines 62-71.
The only problem was the messy notification message.
(Config path being null does not indicate a problem, it simply means that the user has no mirrord config that can be used)

Reply:
Shouldn't we help them create the first one then?

Reply:
Added an action to create a default config, in case config path is null.
Additionally, changed our notifications a bit - all actions are now wrapped in try catch. When action handler throws, a new error notification is fired (with the "get help" links)